packages/workbench/src/browser/workbench-frontend-contribution.ts

```diff
@@ -0,0 +1,71 @@
+/********************************************************************************
+ * Copyright (C) 2017 TypeFox and others.
```

Comment:
@geropl please adjust copyrights to reflect your company

Reply:
Sure, is the form to simply change `Typefox` to `<your company>`